packages-web/maps/src/components/GoogleApi.ts

```diff
@@ -6,13 +6,14 @@ export interface GoogleApiWrapperState {
     alertMessage?: string;
 }
 
-const googleApiWrapper: Function = (script: string) => <P extends GoogleMapsProps>(
-    wrappedComponent: ComponentType<P>
-) => {
+const googleApiWrapper = (script: string) => <P extends GoogleMapsProps>(wrappedComponent: ComponentType<P>) => {
```

Comment:
Is this wrapper still useful? Basically `GoogleMap` is already mostly doing properties mapping, so maybe they can be merged without decreasing code understandability.

Reply:
Alternatively, we can use an existing [library](https://www.npmjs.com/package/google-map-react) (or [more modern](https://www.npmjs.com/package/@react-google-maps/api)) iso wrapping it ourselves. This can make our GoogleMaps code more aligned with Leaflet and definitely will make it more testable.

Reply:
Done :)